### PR TITLE
Adapt MongoDB to support singularity

### PR DESCRIPTION
**Pull request by ImenBizid**
In this PR, we:
- adapt mongo PCA  to support singularity
- add the required variables to make mongo PCA work for both singularity and Docker
- delete the Resume and Pause actions

### CODE REVIEW

File: MongoDB/resources/catalog/Finish_MongoDB.xml [hunk 1]
```diff
@@ -1,25 +1,162 @@
...
+        <info name="task.icon" value="/automation-dashboard/styles/patterns/img/wf-icons/mongodb.png"/>
+        <info name="Documentation" value="PCA/PCAUserGuide.html"/>
+        <info name="NODE_ACCESS_TOKEN" value="$INSTANCE_NAME"/>
+      </genericInformation>
+      <pre>
+        <script>
+          <code language="bash">
+            <![CDATA[
+CLEAN_DATA=$variables_CLEAN_DATA
+if [ "$CLEAN_DATA" = "true" ]; then
```
Comment (fviale) [inline MongoDB/resources/catalog/Finish_MongoDB.xml:32]: I believe here it should be ==

Comment (andrewssobral) [reply MongoDB/resources/catalog/Finish_MongoDB.xml:32]: Yes me too, and also using Bash4 lowercase mode ${CLEAN_DATA,,}

**Comment by andrewssobral** [comment]
@fviale I asked to @ImenBizid to put this PR on hold until I finish all tests on HPE (SLURM+Singularity), surely new fixes will be done after that.

File: MongoDB/resources/catalog/Finish_MongoDB.xml (hunk 1 above)
Comment (ImenBizid) [reply MongoDB/resources/catalog/Finish_MongoDB.xml]: I fixed it. Thanks for the remark

**Review by andrewssobral — APPROVED** [review]

**Comment by andrewssobral** [comment]
@ImenBizid @fviale I'm approving this PR because it's working fine for Docker + AutoML, but not with Singularity. It seems to be a Dockerfile or maybe some Singularity parameters that should be checked. If we need to change something inside the MongoDB PCA service I will create another PR.

**Comment by andrewssobral** [comment]
@ImenBizid @fviale I'm approving this PR because it's working fine for Docker + AutoML. However, with Singularity, the service is UP and RUNNING, but we got Authentication Failed. This is caused because the entrypoint file is not executed after docker-to-singularity image conversion. It seems to be a Dockerfile or maybe some Singularity parameters that should be checked. If we need to change something inside the MongoDB PCA service I will create another PR.

**Pull request merged**